### DISCUSSION TITLE
Repo referenced on youtube

### DISCUSSION THREAD

**Discussion by nkgilley** [kyleskom/NBA-Machine-Learning-Sports-Betting#106] [General]
I thought this was cool, he uses this repo in the video:

https://www.youtube.com/watch?v=IDthta5sUGQ

**Comment by kyleskom** [comment]
Ha yes I did know about that. Feels pretty cool this "I just want to learn the basics of ml" turned into this.
This has also corresponded to the rise in reported issues as well 🙄😂

**Comment by nealmick** [comment]
Great video!
